scss/components/shellbar.scss

```diff
@@ -65,15 +74,65 @@ $block: #{$fd-namespace}-shellbar;
             }
         }
     }
+
+    &__button {
```

Comment:
should button, title/subtitle and logo be split out as sub files?

Reply:
yeah that would make sense - then I can reuse the styles for title in product-menu as well

Reply:
I decided not to do that for now - the naming convention would be thrown off unless we made them mixins/they make sense in the shellbar base file. Thoughts?